db/migrate/20180108234000_add_password_digest_to_user.rb

```diff
@@ -0,0 +1,7 @@
+class AddPasswordDigestToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :password_digest, :string
+
+    User.connection.execute "UPDATE users SET password_digest = identities.password_digest FROM identities WHERE users.id = identities.user_id"
```

Comment:
:'(

Reply:
Meh, it was only my user in the DB when this ran.